mftool.py

```diff
@@ -101,19 +108,7 @@ def get_scheme_quote(self, code, as_json=False):
         """
         code = str(code)
         if self.is_valid_code(code):
-            scheme_info = {}
-            url = self._get_quote_url
-            response = self._session.get(url)
-            data = response.text.split("\n")
-            for scheme_data in data:
-                if code in scheme_data:
-                    scheme = scheme_data.split(";")
-                    scheme_info['scheme_code'] = scheme[0]
-                    scheme_info['scheme_name'] = scheme[3]
-                    scheme_info['last_updated'] = scheme[5].replace("\r", "")
-                    scheme_info['nav'] = scheme[4]
-                    break
-            return self.render_response(scheme_info, as_json)
+            return self.render_response(self.scheme_codes[code], as_json)

```

Comment:
If `self.is_valid_code(code)` is correct, response is fetched from already processed response saved under scheme_code.

This reduces the request time under duplicate request made on line 106.